packages/expo-ui/ios/Modifiers/ViewModifierRegistry.swift

```diff
@@ -115,6 +115,92 @@ internal struct ForegroundColorModifier: ViewModifier {
   }
 }
 
+internal struct ForegroundStyleModifier: ViewModifier {
+  let styleType: String
+  let hierarchicalStyle: String?
+  let color: Color?
+  let gradientColors: [Color]?
+  let gradientStartPoint: UnitPoint?
+  let gradientEndPoint: UnitPoint?
+  let gradientCenter: UnitPoint?
+  let gradientStartRadius: CGFloat?
+  let gradientEndRadius: CGFloat?
```

Comment:
Could we just name these with out the gradient prefix and clean up the bindings? Seems like the only reason we name them like this is because they are optional?